start.go

```diff
@@ -282,16 +119,7 @@ func start(c *cli.Context) {
 	fmt.Fprintf(stderr, "SAM CLI if you update your AWS SAM template.\n")
 	fmt.Fprintf(stderr, "\n")
 
-	log.Fatal(http.ListenAndServe(c.String("host")+":"+c.String("port"), router))
-
-}
+	// Start the HTTP listener
+	log.Fatal(http.ListenAndServe(c.String("host")+":"+c.String("port"), mux.Router()))
```

Comment:
I don't like passing around this mysteriously named object called `c` to every method. We should create a command handling class that has the right properties/methods to retrieve these values.